HumanControls/MenuController.cpp

```diff
@@ -38,14 +38,19 @@ MenuController::MenuController(unsigned int encoderPinClk,
     currentPage = &dashPage;
 }
 
-void MenuController::init()
+void MenuController::init(String status)
```

Comment:
This should be a reference as well. Otherwise it will copy strings and leave memory fragment holes.

Reply:
Done.